eureka-webapp/src/main/webapp/assets/js/eureka.registration.js

```diff
@@ -137,7 +137,14 @@ window.eureka.registration = new function () {
 				},
 				error: function (xhr, err) {
 					$('#passwordChangeFailure').show();
-					$('#passwordErrorMessage').text(xhr.responseText)
+                                        if (xhr.status == "409") {
+                                                $('#passwordErrorMessage').html("Account already exists! You may go ahead and <a href='protected/login'>login</a>.");                                            
+                                        } else if (xhr.status == "400"){
+                                                $('#passwordErrorMessage').html("Please review your profile and complete all required fields.");
+                                        } else {
+                                                $('#passwordErrorMessage').html("Servlet Exception "+xhr.status);
```

Comment:
Check with Akshatha on code she recently committed for a "something went wrong" message.